RedPitaya_emulator/srcbin/emul.c

```diff
@@ -0,0 +1,110 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<stdint.h>
+#include<time.h>
+#include<signal.h>
+
+#include "TCP_API.h"
+
+#define PORT 7538
+
+float x0=80.0;
```

Comment:
And ideally list some settings as well - that could be used as well in the long run to identify the settings / variables / configs / ... that will need to be stored in a dicom.